app/ledger.js

```diff
@@ -724,8 +725,6 @@ var updateLedgerInfo = () => {
   var info = ledgerInfo._internal.paymentInfo
   var now = underscore.now()
 
-  if (!client) return
```

Comment:
@mrose17 says this was here because:
> until `client` is defined, there isn’t any `ledgerInfo` so, yes, all the fields will be undefined.

however, as of https://github.com/brave/browser-laptop/pull/3432/files#diff-9b959b675446ba95e1abe0b81e0f7a9bR133, ledgerInfo does in fact have the `creating` property before `client` is created. 

therefore i think this line above should be removed. if any calls to `updateLedgerInfo` require existence of `client`, they should do that check before calling `updateLedgerInfo`.